code/modules/admin/secrets.dm

```diff
@@ -427,8 +427,7 @@
 				L.fix()
 
 		if("floorlava")
-			var/datum/weather/floor_is_lava/storm = new /datum/weather/floor_is_lava
-			storm.weather_start_up()
+			SSweather.run_weather("the floor is lava")
```

Comment:
Yes, number values with descriptive defines. There is no reason to use strings for this sort of stuff.